README.md

```diff
@@ -54,7 +54,8 @@
 - Use [timeouts](#timeouts) to avoid hanging CI jobs
 - Print [Cypress info](#print-cypress-info) like detected browsers
 - Run [tests nightly](#nightly-tests) or on any schedule
-- Suppress [test summary](#suppress-test-summary)
+- Specify [job summary title](#job-summary-title)
```

Comment:
I like this change. I do find the number of examples is quite overwhelming and identifying all of the supported configurations is really difficult to quickly identify in our current read me. Something that's been on my mind when reviewing this repo in general.